packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -19,33 +20,42 @@ const minify = () => {
     });
 };
 
-const COMMA_CHAR_CODE = 58;
+const removeSpacesBeforeSelector = (selector: parser.Node) => (selector.spaces.before = '');
+const doesCombinatorTypePrecedesSelector = (selector: parser.Node) => {
+  const previousSelector = selector.prev();
 
+  return previousSelector && previousSelector.type === 'combinator';
+};
+const prependNestingTypeToSelector = (selector: parser.Node) => {
+  const { parent } = selector;
+
+  if (parent) {
+    const nesting = parser.nesting({});
+
+    parent.insertBefore(selector, nesting);
+  }
+};
 const parentOrphenedPseudos = plugin('parent-orphened-pseudos', () => {
   return (root) => {
     root.walkRules((rule) => {
-      if (rule.selector.includes(':')) {
```

Comment:
should we keep something like this so we don't parse every selector unnecessarily?

Reply:
`if (selector.type === 'pseudo')` I added this pseudo check. It's same right.

Reply:
not really because we've already parsed the selectors by this point 🤔

Reply:
hmm..so I was thinking of avoiding string checks..I think you mean that we should only pass those selectors to `postcss-selector-parser` which includes `:`, but in this case I think it will be difficult to assign value back to `rule.selector` because `postcss-selector-parser` will only parse those selectors which includes `:`. Have to check.

Actually, I referred this example: https://github.com/postcss/postcss-selector-parser/blob/master/API.md#synchronous-error-example

Reply:
I think I can assign back to `rule.selector` 👍

Reply:
Sorry my bad 😞

Reply:
all good!